src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.csproj

```diff
@@ -7,10 +7,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <ItemGroup>
-    <Compile Include="System.Diagnostics.PerformanceCounter.cs" />
+    <Compile Include="System.Diagnostics.PerformanceCounter.cs">
+      <SubType>Component</SubType>
```

Comment:
Building with VS17 automatically adds it again. msbuild does not add it. I think it gets added by VS because it derives from System.ComponentModel.Component and VS has a tooling for that in the Solution Explorer. So keep it or remove it till the next time?